src/web/src/config.ecpp

```diff
@@ -38,64 +38,378 @@
 #include <fty_common_rest_utils_web.h>
 #include <fty_common_rest_audit_log.h>
 
-// define json serialization objects
-struct Array
-{
-    struct Config
-    {
-        std::string key;
-        std::vector <std::string> value;
-    };
-    Config config;
-};
+/**
+ * XXX: Move that file into the file-system and maybe split it per components.
+ *
+ * fty-rest doesn't seem to have any way to simply drop files into the
+ * file-system, keep this in memory until that can be reworked.
+ */
+static const char* mapping = R"xxx(
```

Comment:
all these should go into some external descriptor file a la dotD, so that we can ship some FTY version, and complete with some IPM2 drops (coming from an etn pkg)